go/gilded_rose.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import "fmt"
+
+type GildedRose struct {
```

Comment:
this struct is not necessary.
It's not the code that goes straight to the point.
This struct can be made in the kata but not in the starter.